api/MatterwikiAPI.js

```diff
@@ -0,0 +1,37 @@
+/* This is the wrapper for the Matterwiki API
+The main function in the wrapper is the .call(endpoint,type,body,token) which returns a
+promise with the response.
+*/
+
+var Promise = require("bluebird");
+var os = require("os");
+var hostname = os.hostname();
+import unirest from "unirest";
```

Comment:
I'm not exactly using `unirest`, tried it initially, but then moved back to the `request` package. As it is, I forgot to remove unirest. Also, `unfetch` seems great! Will check it out.

This is another benefit of using a wrapper, you don't have to change the code in every component. You just change it in the wrapper and everything is taken care of :heart_eyes: